app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyUpdate.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.teleconsultlog.medicinefrequency
+
+import com.spotify.mobius.Next
+import com.spotify.mobius.Update
+import org.simple.clinic.mobius.dispatch
+
+class MedicineFrequencyUpdate : Update<MedicineFrequencyModel, MedicineFrequencyEvent, MedicineFrequencyEffect> {
+  override fun update(model: MedicineFrequencyModel, event: MedicineFrequencyEvent): Next<MedicineFrequencyModel, MedicineFrequencyEffect> {
+    return when (event) {
+      is SaveMedicineFrequencyClicked -> dispatch(SaveMedicineFrequency(model.medicineFrequency))
```

Comment:
Over here we are supposed to pass the event `MedicineFrequency` instead of the model because we are not updating the model `MedicineFrequency` yet.